scripts/strun

```diff
@@ -25,6 +25,10 @@ if __name__ == '__main__':
     try:
         step = Step.from_cmdline(sys.argv[1:])
     except NoDataOnDetectorError:
+        import traceback
+        traceback.print_exc()
         sys.exit(64)
```

Comment:
Nothing to do with this PR, but we should have some documentation in the code here as to the significance of the exit status = 64, as described in painful detail in #2336.

And speaking of #2336, it appears that the work to modify the ASN generator rules to exclude some NRS2 datasets (#2440) has never been implemented? We should do that.

Reply:
Added some documentation.

About the associations, I hear that the new gitbot is about ready. When that happens, a new jira issue should be created linking to #2336.